### PR TITLE
Changes wanReplicationRef config to support republishing

### PR DESCRIPTION
**Pull request by emrahkocaman**

### CODE REVIEW

**Comment by bilalyasar** [comment]
verify

**Comment by mesutcelik** [comment]
:+1:

File: hazelcast/src/main/resources/hazelcast-config-3.5.xsd [hunk 1]
```diff
@@ -1319,6 +1319,7 @@
     <xs:complexType name="wan-replication-ref">
         <xs:sequence>
```
Comment (bilalyasar) [inline hazelcast/src/main/resources/hazelcast-config-3.5.xsd]: i think, this should be `all` instead of `sequence`,  otherwise order will be important,

**Comment by bilalyasar** [comment]
you have to update ConfigXmlGenerator#wanReplicationXmlGenerator

**Comment by emrahkocaman** [comment]
I've moved this PR to https://github.com/hazelcast/hazelcast/pull/5008.

**Pull request closed without merging**